### PR TITLE
Update style of international landing page

### PR DESCRIPTION
**Pull request by richtier**
[this task](https://uktrade.atlassian.net/browse/ED-533)
[this task](https://uktrade.atlassian.net/browse/ED-565)

image

image

### CODE REVIEW

**Comment by richtier** [comment]
Missing links and content will be filled in when they are known

File: enrolment/templates/landing-page-international-form.html [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+						</div>
+					</div>
+				</div>
+
+				<hr class="custom-hr">
+
+				<div class="row-fluid">
+					<div class="span4 checkbox-tc {{ form.full_name.css_classes|default:'input-field-container' }}">
+						{{ form.terms.errors }}
+						<label for="">
```
Comment (monikasulik) [inline enrolment/templates/landing-page-international-form.html]: Shouldn't for be filled in here?

File: enrolment/views.py [hunk 2]
```diff
@@ -31,6 +31,16 @@ def render_to_response(self, context, **response_kwargs):
         return response
 
 
+class HandleBuyerFormSubmitMixin:
+    success_template = 'landing-page-international-success.html'
+    form_class = forms.InternationalBuyerForm
+
+    def form_valid(self, form):
+        data = forms.serialize_international_buyer_forms(form.cleaned_data)
+        api_client.buyer.send_form(data)
```
Comment (monikasulik) [inline enrolment/views.py]: Shouldn't there be a test to check that this api_client method was definitely called? (or is there already one somewhere?)

File: enrolment/templates/landing-page-international-form.html [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+							<input id="" name="{{ form.terms.html_name }}" type="checkbox" class="checkbox pull-left">
```
Comment (monikasulik) [inline enrolment/templates/landing-page-international-form.html]: also here - input id is missing - is that definitely correct?

File: enrolment/templates/landing-page-international-form.html [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+		<h2>Find the best UK exporters</h2>
+		<p>
+			Get access to the UK supply chain. Register to be notified as soon as companies that match your needs are live on the Find a Supplier service.
+		</p>
+
+		<div class="row-fluid">
+			<form action="" method="post">
+				<div class="row-fluid">
+					<div class="span4">
+						<div class="{{ form.full_name.css_classes|default:'input-field-container' }}">
```
Comment (monikasulik) [inline enrolment/templates/landing-page-international-form.html]: Wouldn't it make sense to abstract some of the form html out? Either a form template to extend/include or if this form is too non-standard then maybe a field template that can be included for the fields that are have straightforward markup?

File: enrolment/views.py (hunk 2 above)
Comment (richtier) [inline enrolment/views.py]: right here :) https://github.com/uktrade/directory-ui/pull/263/files#diff-06296d74f202ae56a535678f49bd91f5R251

Comment (monikasulik) [inline enrolment/views.py]: ah sorry, for some reason thought it was asserting some other function was being called.

**Pull request merged**